src/components/Player/Player.jsx

```diff
@@ -38,6 +41,8 @@ const getPlayerSubroute = (info, playerId) => {
       return <ProsPage playerId={playerId} />;
     case 'rankings':
       return <RankingsPage playerId={playerId} />;
+    case 'histograms':
+      return <HistogramsPage playerId={playerId} histogramName={subInfo} />;
```

Comment:
It says in the docs that you shouldn't get params that way due to how react-router is async and doesn't always align with the store. It says you can use mapStateToProps in an example, but all they are doing is pulling the params from the props (which is unnecessary as they already exist as props). https://github.com/reactjs/react-router-redux#how-do-i-access-router-state-in-a-container-component